### ISSUE TITLE
Running on AWS Lambda

### ISSUE THREAD

**Issue by jsclose** [thtrieu/darkflow#915]
I know this is not necessarily an issue, but has anyone been able to use darkflow on AWS lambda? Not sure where else to ask this question. Thanks

**Comment by abagshaw** [comment]
Yes, a while ago I tried but unfortunately Lambda just isn't suited well for this use case. It takes a substantial amount of time for darkflow to load up the models and because Lambda won't keep your function "hot" for long it'll most likely end up unloaded quite frequently resulting in most requests having an unacceptably long latency.

If you're looking for a hosted solution for darkflow you might want to check out [Ciliar](https://ciliar.co) (disclaimer I run it).

Algorithmia is another company trying to make strides in the "serverless ML" area, but in my opinion their cold start latency really kills any benefits of the serverless model.

**Comment by teamtoshare** [comment]
Hi, @jsclose, did you get it working? The docker image in combination with the AWS batch?

**Comment by jsclose** [comment]
@teamtoshare , yes, I was able to write a wrapper around darknet in python that lets you run darknet on a video with custom configuration. I wrote a script that downloads a video from s3, process it with the wrapper, then uploads it back to s3. I created a dockerfile that downloads all the necessary components and then pushed the docker image to amazon ecr which can be used by AWS batch. Lastly, I have a lambda function that is trigger when new videos are uploaded to s3 and submits the job to the aws batch job queue

**Comment by teamtoshare** [comment]
@jsclose, thank you for the update. That's sounds very interesting to me. Did you share this project on GitHub? I couldn't find it in your repo. If you don't mind, I would like to have a look at the script and the dockerfile.

**Comment by teamtoshare** [comment]
@jsclose, sorry to bother again, but would you mind me having a look at the script? Thanks for the response in advance.
